authorization/service/addDevice.go

```diff
@@ -54,11 +54,12 @@ func (s *Service) AddDevice(ctx context.Context, request *pb.AddDeviceRequest) (
 	defer tx.Close()
 
 	owner := request.UserId
-	token, err := grpc_auth.AuthFromMD(ctx, "bearer")
-	if err != nil {
-		uid, err := grpc.ParseOwnerFromJwtToken(s.ownerClaim, token)
-		if err == nil && uid != serviceOwner {
-			owner = uid
+	if owner == "" {
```

Comment:
This will be refactored later? As we discussed owner will be taken only from the token.

Reply:
Yes, it must be done later, because if I remove this it will break tests. We need to develop a mechanism with inserts a custom userId into a token and update the tests, not sure if there is a story for this in monday